gulpfile.ts

```diff
@@ -1,4 +1,6 @@
-import {Gulpclass, Task, SequenceTask, MergedTask} from "gulpclass";
+import { Gulpclass, Task, SequenceTask, MergedTask } from "gulpclass";
+
+declare function require(module: string): any;
```

Comment:
don't think you should do it. `require` should be defined and comes from `@types/node` package. Why did you add it?